src/java/org/apache/cassandra/service/StorageService.java

```diff
@@ -6112,6 +6125,8 @@ public long getClientLargeReadWarnThresholdKB()
     @Override
     public void setClientLargeReadWarnThresholdKB(long threshold)
     {
+        if (threshold < 0)
```

Comment:
nit: If 0 becomes no longer the value for "unlimited", this changes to `<= 0`

Reply:
I don't follow your comment.  0 = disabled

Reply:
we spoke in slack; dumping here

@maedhroz 
> I think what I was getting at is avoiding having special behavior for zero/making the only valid values > 0 ...given the enabled flag is now available. Unless you want to have both the enabled flag and zero as the kill switch for the individual limit types :thinking_face:

@dcapwell 

> yep that was the thinking; to disable all flags (and metrics) enabled=false, to disable specific checks (warn or abort or both) you set to 0. i didn't want to have an enable flag at each check as enable=true is hard as we don't have sensible defaults atm, and what happens if enabled=false and warn/abort are set by users?  it added more user facing compliactions i felt